notNeededPackages.json

```
@@ -3972,6 +3972,12 @@
             "sourceRepoURL": "https://github.com/sindresorhus/wallpaper",
             "asOfVersion": "4.3.0"
         },
+        {
+            "libraryName": "watson-developer-cloud",
+            "typingsPackageName": "watson-developer-cloud",
+            "sourceRepoURL": "https://github.com/watson-developer-cloud/node-sdk",
+            "asOfVersion": "3.0.1"
```

This should probably be `3.0.0`, since that includes the `3.0.0` pre‑releases, which already had this.
```suggestion
            "asOfVersion": "3.0.0"
```

---

Travis took issue with using 3.0.0: `AssertionError [ERR_ASSERTION]: The specified version 3.0.0 of watson-developer-cloud is not on npm.`

https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/596592948

The possible releases here are:
* 3.0.0-beta1
* 3.0.0-rc1
* 3.0.1

---

For reference, I'm not sure if the expectation is to change the string then to something like `3.0.0-beta1` or just leave it as is `3.0.1`.